### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Re-did the install, and found out you do not need to use a Rosetta mode Terminal. But you do need to run xCode in Rosetta mode.

#### Ticket Link
- none

### CODE REVIEW

File: site/content/contribute/mobile/developer-setup/_index.md [hunk 1]
```diff
@@ -108,14 +108,13 @@ Some distributions come with Git preinstalled but you'll most likely have to ins
...
-3. With m1 mac you **may** have to install cocoapods using a Rosetta terminal. Follow step 1 from [this blog post](https://www.courier.com/blog/tips-and-tricks-to-setup-your-apple-m1-for-development/) to install a Rosetta terminal.
-4. In the Rosetta terminal, change to the `mattermost-mobile/ios` directory and run:
+3. In the Rosetta terminal, change to the `mattermost-mobile/ios` directory and run:
 ```sh
 sudo arch -x86_64 gem install ffi
 sudo gem install cocoapods
 arch -x86_64 pod install
 ```
-
+4. If you need to use the xCode app (e.g. to build, sign, and transfer the app to an iOS device), be sure to start it in Rosetta mode.
```
Comment (cwarnermm) [inline site/content/contribute/mobile/developer-setup/_index.md:117]:
```suggestion
4. If you need to use the xCode app (e.g., to build, sign, and transfer the app to an iOS device), be sure to start it in Rosetta mode.
```

**Review by cwarnermm — APPROVED** [review]
Thanks! One suggestion provided inline. Approving for expediency.

**Review by enahum — APPROVED** [review]

File: site/content/contribute/mobile/developer-setup/_index.md [hunk 2]
```diff
@@ -108,14 +108,13 @@ Some distributions come with Git preinstalled but you'll most likely have to ins
...
-3. With m1 mac you **may** have to install cocoapods using a Rosetta terminal. Follow step 1 from [this blog post](https://www.courier.com/blog/tips-and-tricks-to-setup-your-apple-m1-for-development/) to install a Rosetta terminal.
-4. In the Rosetta terminal, change to the `mattermost-mobile/ios` directory and run:
+3. In the Rosetta terminal, change to the `mattermost-mobile/ios` directory and run:
 ```sh
 sudo arch -x86_64 gem install ffi
 sudo gem install cocoapods
 arch -x86_64 pod install
 ```
-
+4. If you need to use the xCode app (e.g., to build, sign, and transfer the app to an iOS device), be sure to start it in Rosetta mode.
```
Comment (enahum) [inline site/content/contribute/mobile/developer-setup/_index.md:117]:
```suggestion
4. If you need to use the Xcode app (e.g., to build, sign, and transfer the app to an iOS device), be sure to start it in Rosetta mode.
```

is actually called `Xcode`

Comment (enahum) [reply site/content/contribute/mobile/developer-setup/_index.md:117]: same in L#2

**Review by enahum — CHANGES_REQUESTED** [review]

File: site/content/contribute/mobile/developer-setup/_index.md (hunk 2 above)
Comment (cpoile) [reply site/content/contribute/mobile/developer-setup/_index.md]: Thanks!

**Review by enahum — APPROVED** [review]

**Comment by enahum** [comment]
awesome thank you

**Pull request merged**